main/debootstrap/APKBUILD

```diff
@@ -1,24 +1,24 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=debootstrap
-pkgver=1.0.96
+pkgver=1.0.97
 pkgrel=0
 pkgdesc="Debian/Ubuntu bootstrap scripts"
 url="https://packages.qa.debian.org/d/debootstrap.html"
 arch="noarch"
-license="GPL-2.0"
+license="MIT"
 options="!check" # upstream doesn't have a test suite
 depends="debian-archive-keyring dpkg tar"
 makedepends=""
 subpackages="$pkgname-doc"
 source="http://ftp.debian.org/debian/pool/main/d/${pkgname}/${pkgname}_${pkgver}.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
 
 package() {
 	cd "$builddir"
 	make DESTDIR="$pkgdir" install
-	install -Dm644 $pkgname.8 \
+	install -D -m644 $pkgname.8 \
```

Comment:
Can you explain why you are making these changes?
If it doesn't make much sense please don't clobber patches with it.

Reply:
@clandmeter Yes. It looks a bit cleaner but probably redundant.
Reverted.

Reply:
The thing is, if we don't set a specific format for these kind of things everyone will apply their own preference making diffs larger than needed.

I know its nitpicking so I'm sorry for that :)

Reply:
Nothing to worry about :) Since there's no specific format your feedback is very much appreciated.